### PR TITLE
Initial version of the policy library documentation

### PR DESCRIPTION
**Pull request by flavius-dinu**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [x] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [ ] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/concepts/policy/README.md [hunk 1]
```diff
@@ -288,6 +288,30 @@ Yes, policy sampling is perfectly safe. Session data may contain some personal i
 
 Last but not least, the policy workbench - including access to previous inputs - is only available to **Spacelift account administrators**.
 
+## Policy library
+
+!!! info
+    This feature is currently in open-beta.
+
+Sometimes, Rego can be hard, especially if you are just starting out with it. The Policy Workbench is great for helping you get policies right, but with the Policy library, we take it up a notch.
```
Comment (jmfontaine) [inline docs/concepts/policy/README.md:296]: I don't think mentioning Rego is the best approach here. I would instead mention OPA, which is likely known to more people and, while less precise, is still correct.

Comment (jmfontaine) [inline docs/concepts/policy/README.md:296]: I would add a link to the [policy workbench section](https://docs.spacelift.io/concepts/policy/#policy-workbench).

Comment (flavius-dinu) [reply docs/concepts/policy/README.md]: Done.

Comment (flavius-dinu) [reply docs/concepts/policy/README.md]: Done.

**Review by eminalemdar — APPROVED** [review]
LGTM

**Pull request merged**